force-app/main/default/classes/JobApplicationREST.cls

```diff
@@ -55,11 +55,11 @@ global with sharing class JobApplicationREST {
         String jobApplicationName = request.requestURI.substring(
```

Comment:
now we store Id in variable jobApplicationName, please rename it

Reply:
Oops:)